tests/id-test.js

```diff
@@ -39,11 +40,12 @@ test('Recreate an B58 String', function (t) {
 
 test('Recreate from a Public Key', function (t) {
   var id = PeerId.createFromPubKey(testId.pubKey)
-  t.ok(id)
+  t.ok(id.pubKey == testId.pubKey)
```

Comment:
that's actually a string comparison so i should use t.equal(a,b)... but i'm switching tape out for mocha atm anyway.